packages/@lwc/engine-dom/src/renderer.ts

```diff
@@ -187,6 +187,14 @@ export function createComment(content: string): Node {
     return document.createComment(content);
 }
 
+export function cloneNode(node: Node, deep: boolean): Node {
+    return node.cloneNode(deep);
+}
+
+export function createFragment(html: string): Node | null {
+    return document.createRange().createContextualFragment(html).firstChild;
```

Comment:
Why `document.createRange().createContextualFragment(html)` instead of `document.createFragment(html)`?

Reply:
IE11

Reply:
It might be worth adding a comment, since it's not obvious why it's needed.

Reply:
I found [this SO post](https://stackoverflow.com/a/25214113) with a good overview of the different options. This method is the best for the broadest browser support.